includes/chart.js

```diff
@@ -336,6 +336,43 @@ function updateEdges(rowID, edgesGroup) {
         // Append the line to the edges group
         edgesGroup.appendChild(line);
 
+        // Add deontic text between second and third nodes
+        if (link.source === 2 && link.target === 3) {
+
+            // Find the row object that corresponds to the current rowID
+            let rowObj = rowValues.find(row => row[0] === rowID);
+
+            // Check if 'deontic' attribute is defined in the array
+            if (rowObj && rowAttributes.indexOf('deontic') !== -1) {
```

Comment:
We don't have to check for `rowAttributes.indexOf('deontic')`: it's always defined in the `const` list of attribute names. We only have to check for the actual value in the `rowObj`, which we do later.